### PR TITLE
Milestone 2: Hit the API

### PR DESCRIPTION
**Pull request by Kingjosh007**
Here is an overview of what I did in this milestone: 

- I created a `api-stuff.js` file where I wrote two functions: **postScore(**data**)** and **getScores( )**
- I updated the code to display the scores from the API call
- I handled the form submission (adding a new user with his score)
- I used **async/await** in my code, following [the requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m2_send_receive_data.md)
- I deployed this branch to Github Pages (see the live link in the README.md file)
- I added a screenshot gif to the README, showing how it works.

### CODE REVIEW

File: src/api-stuff.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+const scoresURL = `${baseURL}games/${myGameId}/scores/`;
+
+async function postScore(data) {
+  const res = await fetch(scoresURL, {
+    method: 'POST',
+    mode: 'cors',
+    headers: { 'Content-type': 'application/json' },
+    body: JSON.stringify(data),
+  });
+  return res.json(); // parses JSON response into native JavaScript objects
```
Comment (dasileker) [inline src/api-stuff.js:12]: - [ ] Please use the Arrow function as the requirements say.

olo

File: src/api-stuff.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+}
+
+async function getScores() {
+  const res = await fetch(scoresURL);
+  return res.json();
+}
```
Comment (dasileker) [inline src/api-stuff.js:18]: - [ ] Also here use the arrow function.

File: src/data.js [hunk 3]
```diff
@@ -1,24 +1,52 @@
...
   lbl.innerHTML = '';
   const codeForData = data.sort((a, b) => b.score - a.score)
     .map((el, i) => `<li id="${idFromIndex(i)}" class="show">
                 <span class="rk">${i + 1}</span>
-                <span class="name">${el.name}</span>
+                <span class="user">${el.user}</span>
                 <span class="score">${el.score}</span>
               </li>`).join('');
   lbl.innerHTML = codeForData;
 }
```
Comment (dasileker) [inline src/data.js:25]: - [ ] Also you should refactor all the functions to the arrow functions.

Comment (Kingjosh007) [reply src/data.js]: Done.

**Pull request merged**